contracts/governance/extensions/GovernorSuperQuorum.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {Governor} from "../Governor.sol";
+import {IGovernorCounting} from "./IGovernorCounting.sol";
+import {SafeCast} from "../../utils/math/SafeCast.sol";
+import {Checkpoints} from "../../utils/structs/Checkpoints.sol";
+
+/**
+ * @dev Extension of {Governor} with a super quorum. Proposals that meet the super quorum can be executed
+ * earlier than the proposal deadline. Counting modules that want to use this extension must implement
+ * {IGovernorCounting}.
+ *
+ * NOTE: It's up to developers to implement `superQuorum` and validate it against `quorum`.
+ */
+abstract contract GovernorSuperQuorum is Governor {
+    /**
+     * @dev Minimum number of cast votes required for a proposal to reach super quorum. Only FOR votes are counted towards
+     * the super quorum. Once the super quorum is reached, an active proposal can proceed to the next state without waiting
+     * for the proposal deadline.
+     *
+     * NOTE: The `timepoint` parameter corresponds to the snapshot used for counting vote. This allows to scale the
+     * quorum depending on values such as the totalSupply of a token at this timepoint (see {ERC20Votes}).
+     */
```

Comment:
Following my previous comment, I think this is clearer in regards to what the developer must care and what the consequences are

```suggestion
     *
     * NOTE: Make sure the value specified for the super quorum is greater than {quorum}, otherwise, it
     * may be possible to pass a proposal with less votes than the default quorum.
     */
```

Reply:
Addressing in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5492/commits/bb9a98976fca6bf33b7c72c543e9c7ef9fef2925